osquery/events/darwin/openbsm.cpp

```diff
@@ -19,14 +19,14 @@ namespace osquery {
 
 /// The OpenBSM subsystem may have a performance impact on the system.
 FLAG(bool,
-     disable_openbsm,
+     disable_audit,
```

Comment:
This flag is defined in the audit implementation too? Perhaps we should unify the definition and protect against duplicate code and generated docs.